tests/utils/test_testing.py

```diff
@@ -68,46 +48,13 @@ def test_assert_elements_are_identical_metadata(full_sdata):
 
     to_iter = list(copied.gen_elements())
     for _, element_name, element in to_iter:
-        if get_model(element) in (Image2DModel, Image3DModel):
```

Comment:
Here I was testing if `assert_elements_are_identical` is able to compare not just the data but also the metadata. Removing them reduces the strength of the test. The code is quite ugly, but I'd rather simplify/keep it like this than remove it, since it's used only in the tests.

Reply:
Added it back, but without duplication of asserts and now the tests are run first with elements being equal and then with elements not being equal.